recipes/gc/all/conanfile.py

```diff
@@ -0,0 +1,144 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanException
+import os
+
+
+class GcConan(ConanFile):
+    name = "gc"
```

Comment:
```suggestion
    name = "bdwgc"
```

https://github.com/microsoft/vcpkg/tree/master/ports/bdwgc

Reply:
Done